profiles/schema.py

```diff
@@ -0,0 +1,97 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+    vocabulary = graphene.String()
+    code = graphene.String()
+    label = graphene.String()
+
+    def resolve_vocabulary(self, info, **kwargs):
+        print(dir(self.vocabulary))
+        return self.vocabulary.prefix
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+        exclude = ("id", "user")
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+    def resolve_concepts_of_interest(self, info, **kwargs):
+        return self.concepts_of_interest.all()
+
+    def resolve_divisions_of_interest(self, info, **kwargs):
+        return self.divisions_of_interest.all()
+
+
+Language = graphene.Enum("Language", [(l[1], l[0]) for l in settings.LANGUAGES])
+ContactMethod = graphene.Enum(
+    "ContactMethod", [(cm[1], cm[0]) for cm in settings.CONTACT_METHODS]
+)
+
+
+class ProfileInput(graphene.InputObjectType):
+    nickname = graphene.String()
+    image = graphene.String()
+    email = graphene.String()
+    phone = graphene.String()
+    language = Language()
+    contact_method = ContactMethod()
+    concepts_of_interest = graphene.List(graphene.String)
+    divisions_of_interest = graphene.List(graphene.String)
+
+
+class CreateProfile(graphene.Mutation):
+    class Arguments:
+        profile = ProfileInput(required=True)
+
+    ok = graphene.Boolean()
+    profile = graphene.Field(ProfileType)
+
+    @login_required
+    def mutate(self, info, **kwargs):
+        profile_data = kwargs.pop("profile")
+        concepts_of_interest = profile_data.pop("concepts_of_interest", None)
```

Comment:
`NoneType` is not iterable, your loops a couple lines down will fail, if this will resolve to `None`. Pass empty list as default.